Vagrantfile

```diff
@@ -57,6 +57,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     bodhi.vm.provider :libvirt do |domain|
         # Season to taste
         domain.cpus = 4
+        domain.cpu_mode = "host-passthrough"
```

Comment:
I was recently reading a [blog post](https://www.berrange.com/posts/2018/06/29/cpu-model-configuration-for-qemu-kvm-on-x86-hosts/) where it was suggested that this was the ideal CPU mode to use if you don't need live migration. Since Vagrant isn't really for production use, I'd say we don't need live migration ☺